homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/games.html

```diff
@@ -0,0 +1,296 @@
+                            <p>Big changes come to the Overwatch formula in this sequel...</p>
+                        </div>
+                    </div>
+                    <div class="game_description">
+                        <dl class="game_details">
+                            <div>
+                                <dt>Genre:</dt>
+                                <dd>Shooter</dd>
+                            </div>
+                            <div>
+                                <dt>Platform:</dt>
+                                <dd>PC (Windows)</dd>
+                            </div>
+                            <div>
+                                <dt>Publisher:</dt>
+                                <dd>Activision Blizzard King</dd>
+                            </div>
+                            <div>
+                                <dt>Developer:</dt>
+                                <dd>Blizzard Entertainment</dd>
+                            </div>
+                            <div>
+                                <dt>Release Date:</dt>
+                                <dd>2022-10-04</dd>
+                            </div>
+                        </dl>
+                    </div>
+                </li>
+                <li class="game_card">
+                    <div class="game_overview">
+                        <img src="./images/Overwatch2.svg" alt="Overwatch2">
+                        <div class="game_info">
+                            <h3>Overwatch 2</h3>
+                            <p>Big changes come to the Overwatch formula in this sequel...</p>
+                        </div>
+                    </div>
+                    <div class="game_description">
+                        <dl class="game_details">
+                            <div>
+                                <dt>Genre:</dt>
+                                <dd>Shooter</dd>
+                            </div>
+                            <div>
+                                <dt>Platform:</dt>
+                                <dd>PC (Windows)</dd>
+                            </div>
+                            <div>
+                                <dt>Publisher:</dt>
+                                <dd>Activision Blizzard King</dd>
+                            </div>
+                            <div>
+                                <dt>Developer:</dt>
+                                <dd>Blizzard Entertainment</dd>
+                            </div>
+                            <div>
+                                <dt>Release Date:</dt>
+                                <dd>2022-10-04</dd>
+                            </div>
+                        </dl>
+                    </div>
+                </li>
+                <li class="game_card">
+                    <div class="game_overview">
+                        <img src="./images/Overwatch2.svg" alt="Overwatch2">
+                        <div class="game__info">
```

Comment:
forget to change, please fix in the next task 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/497ac9eb-7d08-4906-b97b-1a9466a72a44)